plugins/kubecarrier.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kubecarrier
+spec:
+  version: v0.1.0
+  shortDescription: KubeCarrier installation management and control
+  description: |
+    The kubecarrier plugin for installing, upgrading and performing management
+    tasks in the Kubecarrier system.
+
+    KubeCarrier is an open source system for managing applications and services
```

Comment:
Being open source is pretty much a requirement for plugins in krew. There is usually no reason to stress this.